opensir/models/model.py

```diff
@@ -161,9 +167,9 @@ def fit(self, t_obs, n_i_obs, population, fit_index=None):
         model parameters consistent with True entries in the fit_index list.
 
         Args:
-            t_obs (np.array): Vector of days corresponding to the observations
+            t_obs (numpy.ndarray or list): Vector of days corresponding to the observations
```

Comment:
I think there's no real benefit of passing a list here.

The upper layer has always control of whatever is passed to this function and having implicit list <-> np.array implicit conversion adds overhead.

In practice if this data comes from a CSV file, then  it's very likely that it's already a numpy array.

Reply:
The user is likely to do tedious pre-processing, so I agree to enforce np.ndarray.

However, as the index can be integer, string or datetime, it is very likely that the user will process that separately from the numbers of infected, or the state variable to fit against.